nixos/modules/services/web-apps/restya-board.nix

```diff
@@ -305,9 +305,9 @@ in
         mkdir -m 0750 -p "${cfg.dataDir}/client/img"
         cp -r "${pkgs.restya-board}/media/"* "${cfg.dataDir}/media"
         cp -r "${pkgs.restya-board}/client/img/"* "${cfg.dataDir}/client/img"
-        chown "${cfg.user}"."${cfg.group}" "${cfg.dataDir}"
-        chown -R "${cfg.user}"."${cfg.group}" "${cfg.dataDir}/media"
-        chown -R "${cfg.user}"."${cfg.group}" "${cfg.dataDir}/client/img"
+        chown "${cfg.user}":"${cfg.group}" "${cfg.dataDir}"
```

Comment:
What is the purpose of `.` -> `:`?

Reply:
I don't really know where `.` comes from, but `:` is the only correct syntax as stated by the manpage.

Reply:
Fair enough. `:` is legacy, so OK.